mitodl/ocw-studio
# upgrade eslint config

#### What are the relevant tickets?

none, just a follow-up after #252 

#### What's this PR do?

This upgrades our eslint config to a version that I just pushed today which now includes proper configuration for linting react hooks usage.

#### How should this be manually tested?

build should pass, changes should make sense.

You could try introduced a change that violates the rules of hooks and confirming that you get a linting error too if you wanted.
